### DISCUSSION TITLE
Is there a way to edit the sequence of the context menu items of TST?

### DISCUSSION THREAD

**Discussion by bege10** [piroor/treestyletab#3630] [Q&A] [answered]
I edit the sequence of the items of the page context menu with a CSS code.
My attempts to do the same with the context menu of the TST tabs failed. Example (The code contains an entry for every menu item):
```
#treestyletab_piro_sakura_ne_jp-menuitem-_context_duplicateTab{
		-order: 3 !important;
}
```
Is there a special way needed to accomplish that for the TST context menu or is it not possible at all?

**Comment by irvinm** [comment]
I got it to work with:

```
*[id^="treestyletab_piro_sakura_ne_jp-menuitem-_context_duplicateTab"] {
	order: 3 !important;
}
```

![image](https://github.com/user-attachments/assets/d704f113-0cfc-460c-a9e2-bf1f267ce5eb)

**Reply by bege10** [reply, marked as answer]
> I got it to work with:

Thank you very much! Your answer made me see my mistake: the dash in front of `order`. This is not in my CSS code for page context menu. That's why it works there and did not work for the tab context menu.

So this code will work also:
```
#treestyletab_piro_sakura_ne_jp-menuitem-_context_duplicateTab {
		order: 3 !important;
}
```
